ethereum/eip712/eip712_test.go

```diff
@@ -1,30 +1,38 @@
 package eip712_test
 
 import (
+	"bytes"
+	"fmt"
+	goMath "math"
+	"math/big"
+	"reflect"
 	"testing"
 
 	"cosmossdk.io/math"
```

Comment:
Nit: refactor imports names to be consistent with codebase
```suggestion
	"math"
	"math/big"
	"reflect"
	"testing"

	sdkmath "cosmossdk.io/math"
```